update pricing doc for crons (#8773)
getsentry/sentry-docs: src/docs/product/accounts/pricing.mdx

### Per-Category Pricing

> If you'd like to buy either prepaid or on-demand data on top of what's included in your plan, you can buy as much as you think you'll need per data category. [Errors](#errors-pricing), [Performance](#performance-pricing), [Replays](#replays-pricing), [Cron Monitors](#cron-monitors-pricing), and [Attachments](#attachments-pricing), each have their own unique pricing structures, as listed in the tables below:
> 

Reviewer: ```suggestion
If you'd like to buy either prepaid or on-demand data on top of what's included in your plan, you can buy as much as you think you'll need per data category. [Errors](#errors-pricing), [Performance](#performance-pricing), [Replays](#replays-pricing), [Cron Monitors](#cron-monitors-pricing), and [Attachments](#attachments-pricing). Each have their own unique pricing structures, as listed in the tables below:

```

<Note>

> Please note that all per-category pricing listed below have been rounded to the nearest millionth.

Reviewer: ```suggestion
Please note that all per-category prices listed below have been rounded to the nearest millionth.
```

| >25k-100k | 25k | $0.002850 | $0.003705 |
| >100k-1M | 100k | $0.002565 | $0.003335 |

> #### Cron Monitors Pricing
> 
> Monitors for Crons cannot be pre-purchased. Crons are available to use and purchase through a pre-determined on-demand budget that can be shared across all event types or defined per category.
> 

Reviewer: ```suggestion
#### Cron Monitors Pricing

Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a predetermined on-demand budget that can be shared across all event types or defined per category.

```
